deploy/README.md

```diff
@@ -0,0 +1,31 @@
+# Basic Deployment
+
+## Tools
+
+1. [Capistrano](http://capistranorb.com/)
+2. [Ansible](https://www.ansible.com/)
+
+## Servers
+
+1. [Nginx](https://nginx.org)
+2. [Puma](http://puma.io/)
+
+## Managers
+
+1. [RVM](https://rvm.io/)
+2. [NVM](https://github.com/creationix/nvm)
```

Comment:
actually, i don't see a difference between rbenv and rvm, so if you provide any reason why should be rbenv or rvm is worse then rbenv, i will gladly replace rvm with rbenv